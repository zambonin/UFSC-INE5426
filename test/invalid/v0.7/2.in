int a
float b
bool fun f(int a, int b)
bool fun f(){
ret false
}
