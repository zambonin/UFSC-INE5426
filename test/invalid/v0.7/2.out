[Line 6] semantic error: re-definition of function f
[Line 7] semantic error: function f is declared but never defined
